src/gpu/intel/jit/conv/config.cpp

```diff
@@ -903,9 +903,7 @@ bool data_types_ok(
     auto *device_info = compute_engine->device_info();
     if (prb.is_f64_accumulator() && !device_info->has_native(data_type::f64))
         return false;
-    if (is_fp8
-            && !(utils::one_of(hw, ngen::HW::XeHPC) && hw.systolic_support()))
-        return false;
+    if (is_fp8 && !(hw > ngen::HW::Xe2 && hw.systolic_support())) return false;
```

Comment:
This would disable fp8 for XeHPC/Xe2. I think it should be `hw >= ngen::HW::XeHPC`